test/services/dissolution/dissolution.service.test.ts

```diff
@@ -81,14 +90,31 @@ describe('DissolutionService', () => {
     })
   })
 
-  it('should call dissolution api client to approve the dissolution for the provided email', async () => {
-    const email: string = 'test@email.com'
-    const body: DissolutionPatchRequest = generateDissolutionPatchRequest()
+  describe('approveDissolution', () => {
+    it('should call dissolution api client to approve the dissolution for the provided email', async () => {
+      const email: string = 'test@email.com'
+      const body: DissolutionPatchRequest = generateDissolutionPatchRequest()
 
-    when(mapper.mapToDissolutionPatchRequest(email)).thenReturn(body)
+      when(mapper.mapToDissolutionPatchRequest(email)).thenReturn(body)
```

Comment:
Do we need these extra spaces?

Reply:
The tests were missing the describe block, so by introducing it everything has been indented one extra tab

Reply:
Ahhh ok makes sense